README.md

```diff
@@ -252,7 +252,7 @@ Options:
 * Process science data and stage it for Cumulus:
 
         # Source the AWS profile (once) before running 'process'-- use 'default' or a named profile
-        $ source scripts/env.sh default
+        $ source metgenc-env.sh default
```

Comment:
Does the pyproject.toml change create a symlink? (Looking for documentation on this feature but haven't found it yet)